lib/components/form/date-time-modal.tsx

```diff
@@ -32,6 +32,15 @@ export const DepartArriveTypeMap: Record<
   NOW: 'DURATION'
 }
 
+export const DepartArriveDirectionMap: Record<
```

Comment:
Maybe rename to `DepartArriveDefaultSortMap` or something like this, to clarify it is a default setting?